ui-core/src/surveyjs/medications-question.ts

```diff
@@ -6,7 +6,7 @@ ComponentCollection.Instance.add({
   name: 'medications',
   title: 'Medications',
   questionJSON: {
-    type: 'multiple-combobox',
+    type: 'multiplecombobox',
```

Comment:
what's the reason for this change?

Reply:
I had originally started to go down the path of making MultipleCombobox a selectable question type in the question designer, and the dash was causing problems for some of the templating we use for question types. I left it like this because it's consistent with other question type naming and will also make our lives slightly easier if we ever introduce it as a visible feature.